hotshot-state-prover/src/bin/state-prover.rs

```diff
@@ -47,7 +47,7 @@ struct Args {
     l1_provider: Url,
 
     /// Address of LightClient contract on layer 1.
-    #[clap(long, env = "ESPRESSO_SEQUENCER_LIGHTCLIENT_ADDRESS")]
+    #[clap(long, env = "ESPRESSO_SEQUENCER_LIGHT_CLIENT_PROXY_ADDRESS")]
```

Comment:
@alxiong while I do like the new name because it's more consistent than the current one, this is a breaking change for deployments. It would be better to support the old variable and issue a depreciation warning for a while.

cc @rob-maron @Ancient123 @babdor for visibility about the change of name of the env var.

Reply:
is this a breaking change tho? 
previously we set `ESPRESSO_SEQUENCER_LIGHTCLIENT_ADDRESS=$ESPRESSO_SEQUENCER_LIGHT_CLIENT_PROXY_ADDRESS` in the `.env`

it was just a convenient alias as I assume. we are not actually ever gonna use the lightclient alone (without the proxy) iiuc.